main.cpp

```diff
@@ -398,16 +398,12 @@ int main (int argc, char **argv)
 				break;
 			default:
 				{
-#ifndef _TEST
-#ifdef _WINDOWS
-					printf ("Unrecognized option %s\n", argv[curr_arg]);
+					fprintf (stderr,"Unrecognized option %s\n", argv[curr_arg]);

```

Comment:
Make sure the spacing is consistent! (Just add a space between `stderr,` and `"Unrecognized` and you should be good!)